packages/manager-react-components/src/components/breadcrumb/__tests__/__snapshots__/Breadcrumb.spec.tsx.snap

```diff
@@ -0,0 +1,140 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
Since you split the snapshot tests into a different file, this file is no longer needed. Can you remove it?